iset.mm

```diff
@@ -115023,6 +115023,24 @@ Infinite sums (cont.)
       UUASUVTXGXNYTFEVBWEVCCUSUTYFYDKUVJYPUGWNUVRXEXHXFXIXJXK $.
   $}
 
+  ${
+    $d k n $.
+    $( The sum of the reciprocals of the triangle numbers converge to two.
```

Comment:
The sum ... is

Reply:
To be honest I don't really know how one is supposed to say this in the case of an infinite sum. Maybe "the sum ... is" works for infinite but I'd most naturally reach for that in the finite case.

Reply:
There is that, but there is also the grammatical problem, so maybe "the partial sumS ... converge" or at least "the sumS ... converge".